### DISCUSSION TITLE
Laravel Echo cannot be found. Can I use the regular Livewire Echo integration in filamnet?

### DISCUSSION THREAD

**Discussion by StaffCollab** [filamentphp/filament#10592] [Help]
I'm getting the error
`Laravel Echo cannot be found`

It looks like filament loads livewire before listening to echo
It is odd to me, as if I got it correctly, filament is supposed to offer some echo functionality on notifications.

Does anyone have prior knowledge on this issue? 
Please clarify if possible.

Thanks.

**Comment by raonikhil** [comment]
@StaffCollab having the same exact issue, were you able to figure this out 🙏🏼

**Reply by StaffCollab** [reply]
So the answer is that you cannot use Echo in filament.
I have no idea how @danharrin is using it for the database notifications.

There is a straightforward solution if you just need it on one page: you can just re-import your @vite() within that blade file, but it is not very optimal since it will re-import stuff that has already been loaded (you might try to just re-instantiate Echo there...).

**Reply by danharrin** [reply]
Echo comes pre-installed so I'm unsure what you are finding challenging. Please open an issue with a reproduction repo and I will investigate, discussions like this do not help us solve bugs

**Reply by raonikhil** [reply]
@danharrin added here https://github.com/filamentphp/filament/issues/11332
